### ISSUE TITLE
Adjustment of volume via Fn key

### ISSUE THREAD

**Issue by janos-r** [linuxmint/cinnamon#10715]
```
 * Cinnamon version (cinnamon --version)
  5.2.7

 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
 Mint 20.3 una

 * Graphics hardware *and* driver used
 Intel UHD Graphics 630

 * 32 or 64 bit
 64 bit

 * Attach ~/.xsession-errors, or /var/log/syslog
 ```

**Issue**
Adjustment of volume via Fn key is inconsistent

**Steps to reproduce**
1) hover with the mouse over the volume icon in the bottom right
2) scroll with the mouse wheel once down - indication changes for example from 35 to 30
3) use Fn+volUp function key

**Expected behavior**
You would expect it to go back from 30 to 35.
But instead it once repeats the last action, which is it goes down from 30 to 25 (even though you press volUp).

**Other information**
This happens only with this order. Mouse down, key up -> goes down on first press, than continues up as expected.
When I go mouse up, key down -> it goes down as expected on first press.
This was confirmed by other users on the IRC channel. Only that some said that it happened in both directions for them.

**Comment by ItzSwirlz** [comment]
> * Attach ~/.xsession-errors, or /var/log/syslog

Can you just attach this via a pastebin or something? :)

**Comment by DirkHaar** [comment]
Let me guess - you haven't set your "Amplification" value in the Sound Settings, tab Settings, to 100%.
If you haven't, you see changes in the app and via Fn-keys that are strange at first sight, but mathematically correct
(one refers to hardware, one to software settings). I was quite astonished that I never noticed that until now.

**Comment by splitowo** [comment]
Duplicate #10269 
The fix has been merged and should be released with cinnamon 5.4.0 (Mint 21)
